openstack-tripleo-ui-deps.spec

```diff
@@ -25,7 +25,6 @@ ExclusiveArch: x86_64
 BuildRequires:  nodejs
 BuildRequires:  git
 Requires:       %{sname}-babel = %{version}-%{release}
-Requires:       phantomjs >= 1.9.7
```

Comment:
It was removed in [this commit](https://github.com/openstack/tripleo-ui/commit/0376dfe55f487abcec517ea1ec512f58a5bd27ec).